projects/extension/build.py

```diff
@@ -648,7 +648,7 @@ def docker_build() -> None:
 def docker_run() -> None:
     cmd = " ".join(
         [
-            "docker run -d --name pgai-ext -p 127.0.0.1:5432:5432 -e POSTGRES_HOST_AUTH_METHOD=trust",
+            "docker run -d --name pgai-ext --network host -p 127.0.0.1:5432:5432 -e POSTGRES_HOST_AUTH_METHOD=trust",
```

Comment:
Why was this change necessary?

Reply:
because the ollama container and the container in which we run tests, otherwise cannot talk to each other.
The alternative would be to run both in a docker-compose or otherwise share a network. But I felt like this was simpler and should also help for local development if you run ollama separately.